### PR TITLE
Add remove keys

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3964

### CODE REVIEW

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@BillSproule can you review?

File: SharePoint/SharePointOnline/use-silent-account-configuration.md [hunk 1]
```diff
@@ -97,10 +97,12 @@ New-ItemProperty -Path $DiskSizeregistryPath -Name $TenantGUID -Value '102400' -
 
 SilentAccountConfig creates a SilentBusinessConfigCompleted registry entry once SilentAccountConfig has successfully provisioned the user in OneDrive.exe. This prevents SilentAccountConfig from reprovisioning the user in OneDrive.exe if the user manually stops syncing.
 
-If SilentAccountConfig has successfully completed on a computer you're going to use to build a Windows deployment image (for example, SysPrep), you need to ensure this registry key is removed before you prepare your image. You can do so by running the following command:
+If SilentAccountConfig has successfully completed on a computer you're going to use to build a Windows deployment image (for example, SysPrep), you need to ensure this registry key and also ClientEverSignedIn and PersonalUnlinkedTimeStamp registry keys are removed before you prepare your image. You can do so by running the following command:
```
Comment (BillSproule) [inline SharePoint/SharePointOnline/use-silent-account-configuration.md:100]: need add an S to "command:" since there are 3 commands now.

**Review by BillSproule — CHANGES_REQUESTED** [review]
need to add an S, looks good otherwise.

**Comment by get-itips** [comment]
Thanks @BillSproule

**Review by BillSproule — APPROVED** [review]
Thanks!

**Review by BillSproule — APPROVED** [review]
plural case updates look good. Thanks.

**Comment by scanum** [comment]
Hi @BillSproule and @MikePlumleyMSFT  hope all is well. Could you please help us merge this PR? Thanks!

**Pull request merged**